tools/scripts/seed-exams/create-exams.js

```diff
@@ -0,0 +1,76 @@
+import { readFileSync } from 'fs';
+import path, { join } from 'path';
+import { fileURLToPath } from 'url';
+import dotenv from 'dotenv';
+import yaml from 'js-yaml';
+import { MongoClient, ObjectId } from 'mongodb';
+
+import { validateExamSchema } from './exam-schema.js';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+dotenv.config({ path: path.resolve(__dirname, '../../../.env') });
+const { MONGOHQ_URL } = process.env;
+
+// Only these will be added to or updated in the database
+const examFilenames = [
+  'foundational-c-sharp-with-microsoft-certification-exam.yml',
+  'example-certification-exam.yml'
+];
+
+const client = new MongoClient(MONGOHQ_URL, { useUnifiedTopology: true });
+
+const db = client.db('freecodecamp');
+const exams = db.collection('Exam');
```

Comment:
Aren't collection names traditionally plural??

Reply:
MongoDB [say "no"](https://www.mongodb.com/community/forums/t/does-mongodb-have-a-naming-convention-for-collection-and-field-naming/11806/2), but opinionated people [say "yes"](https://stackoverflow.com/questions/5916080/what-are-naming-conventions-for-mongodb)

Personally, I like the idea of following the latter conventions. I constantly have to check if our collection is `user` or `users`.

Reply:
It makes sense. It's a collection of exams, not a collection of exam.

Reply:
I think I'm gonna lose this one though - all of our existing collections are singular ):

Reply:
> Mongoose also automatically pluralises the collection by default.

Trips me up every time I use it :(

Back on topic: I don't have a huge preference, but I think we should be consistent. If we want to rename things, that's fine, but all at once.

Reply:
It's the Exam Collection

Reply:
So, we have `sessions`, `user`, `AccessToken` and `UserToken` already. Name it what you want :man_shrugging:

Reply:
I don't really care what the name is, so if you all think it's better as `exams` or something - let me know.